src/ExplorerRoutes.tsx

```diff
@@ -12,22 +12,17 @@ import BlocksPage from "./pages/Blocks/Index";
 import ValidatorsPage from "./pages/Validators/Index";
 import ValidatorPage from "./pages/Validator";
 import AnalyticsPage from "./pages/Analytics/Index";
-import {useGetInDevMode} from "./api/hooks/useGetInDevMode";
 
 export default function ExplorerRoutes() {
-  const inDev = useGetInDevMode();
-
   return (
     <ExplorerLayout>
       <Routes>
         <Route path="/" element={<LandingPage />} />
         <Route path="/transactions" element={<TransactionsPage />} />
         <Route path="/validators" element={<ValidatorsPage />} />
-        {inDev ? (
-          <Route path="/validator">
-            <Route path=":address" element={<ValidatorPage />} />
-          </Route>
-        ) : null}
```

Comment:
no need to have this `inDev` logic check here cuz we already gated one layer up in validators table row click.